src/errors.js

```diff
@@ -31,6 +31,12 @@ export class RemoteTempInstallNotSupported extends WebExtError {
   }
 }
 
+export class UsageError extends WebExtError {
```

Comment:
this needs a comment at the top summarizing that this error class is for something that the developer will be able to fix when they see it in the console